std_msgs/Header header

bool has_converged
float32 matching_error
float32 inlier_fraction
geometry_msgs/Pose relative_pose

std_msgs/String[] prediction_labels
geometry_msgs/Pose[] prediction_errors